### DISCUSSION TITLE
ib_insync maintainer status

### DISCUSSION THREAD

**Discussion by jlixfeld** [brndnmtthws/thetagang#382] [General]
Hey there,

Not sure if you've [heard](https://groups.io/g/twsapi/topic/question_about_ib_insync/104781778).  Things are a bit in flux at the moment while the community tries to find its footing.  Thought you'd want to know.

**Comment by brndnmtthws** [comment]
Oh, bizarre. It's quite sad that IBKR is unwilling to invest in providing a decent API and tooling, this really shouldn't be left up to random kind folks on the Internet.

**Comment by brndnmtthws** [comment]
Switched to ib_async in https://github.com/brndnmtthws/thetagang/pull/478
